examples/gno.land/r/demo/bytebeat/bytebeat_test.gno

```diff
@@ -0,0 +1,12 @@
+package bytebeat
+
+import (
+	"testing"
+)
+
+func TestByteBeat(t *testing.T) {
+	s := Render("")
+	if len(s) != 128194 {
```

Comment:
very dependent on the format, don't you think?

what about `strings.Contains(s, data)`?

Reply:
Yep, this is fixed now (fixed a bit ago, not sure which commit).